frontend/src/components/VersionSelector/VersionSelector.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { P } from "@maykin-ui/admin-ui";
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { VersionSelector as VersionSelectorComponent } from "./VersionSelector.tsx";
+
+const meta: Meta<typeof VersionSelectorComponent> = {
+  title: "Components/VersionSelector",
+  component: VersionSelectorComponent,
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const VersionSelector: Story = {
+  args: {
+    versions: [
+      { label: "Version 142", id: "142" },
+      { label: "Current Version 143", id: "143", isCurrent: true },
+      { label: "Concept Version 144", id: "144", isConcept: true },
+      { label: "Version 145", id: "145" },
+      { label: "Version 146", id: "146" },
+    ],
+    content: [
```

Comment:
I get the idea but I don't think we'll have access to the detailed information of each version when rendering this so we can't approach it like tabs.

I think this component should not have child content but instead should have a callback, something like "onVersionSelect" that gets called with the given version information (see details endpoint for the schema).